js-foundations.js

```diff
@@ -2,22 +2,11 @@
 // Coding practical questions
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
-  if (num % 3 === 0) {
-    return num + " is divisible by three"
-  } else {
-    return num + " is not divisble by three"
-  }
-}
-console.log(evenlyDistrubed(15))
-console.log(evenlyDistrubed(3))
-console.log(evenlyDistrubed(385))
-console.log(evenlyDistrubed(-7))
+
 
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
-var helloMe = {firstName:"Stephen", lastName:"Bilham", height:"5'9", eyeColor: "brown"}
-console.log("Hello my name is " +helloMe.firstName + " " + helloMe.lastName + ". I am " + helloMe.height + " and the color of my eyes are " + helloMe.eyeColor + ".")
 
```

Comment:
Looks good, you can have the function as part of your object to make your code a little cleaner. And don't forget string interpolation!